install.bat

```diff
@@ -8,10 +8,6 @@ rem This file install.bat can freely used, modified and redistributed, as
 rem long as credit to the author is kept intact. Please send any feedback,
 rem issues or improvements to decalage at laposte.net.
 

```

Comment:
For install.bat, I prefer to keep the changelog inline, as it is not specific to olefile. I'll accept it and restore the previous version, as I can't merge part of the PR.